outside_docker/run_phast.sh

```diff
@@ -1,22 +1,21 @@
 #!/bin/bash
+
 #./scripts/outside_docker/split_count_concat_maf.sh -s subset -i ./scripts/inside_docker/ -o ./chr5_test_mafs/ -c ./output/peak_maf_size.txt -t ./output/concat.maf
 
 main () {
     local command=$1
 
-
-    #start_docker_image 
+    #start_docker_image
     local phast_container='my_phast'
     local script_source='./scripts/maf_splitter/inside_docker/phast_scripts.sh'
     source ./scripts/maf_splitter/outside_docker/docker_utils.sh #contains start_docker_image and d_wrap
 
-    start_docker_image $phast_container "e3d0affb583b" 
+    start_docker_image $phast_container "e3d0affb583b"
```

Comment:
There are a few places where an image is referenced by its hash instead of a tag. I'd switch your image generation into using image tags and container names so you can depend on them locally.